airbyte-cdk/python/airbyte_cdk/sources/abstract_source.py

```diff
@@ -142,7 +150,7 @@ def _read_stream(
         connector_state: MutableMapping[str, Any],
         internal_config: InternalConfig,
     ) -> Iterator[AirbyteMessage]:
-
+        self._apply_log_level_to_stream_logger(logger, stream_instance)
```

Comment:
should this be applied on line https://github.com/airbytehq/airbyte/pull/14521/files#diff-51c861828e0a614ef6ee21390f3ba17d7138549f0575051fdb24868ee05eabbbR106 just in case?

Reply:
I think we should be in the clear because the logger passed into the `read()` method is always coming from `entrypoint()` which is the source level logger and is already set.

Although I'm gonna move the debug statement you mentioned down a level to the stream level since it makes more sense for it to be logged by the stream logger instead of source logger